src/lib/theme/variables.ts

```diff
@@ -0,0 +1,231 @@
+import { toColor } from './utilities';
+
+export type Variable = keyof typeof variables;
+
+const light: Record<CSSVariable, RGB> = {};
+const dark: Record<CSSVariable, RGB> = {};
+
+export const variables = {
+  '--color-primary': {
+    light: 'primary',
+    dark: 'offWhite',
+  },
+  '--color-secondary': {
+    light: 'secondary',
+    dark: 'offBlack',
+  },
+  '--color-inverse': {
+    light: 'black',
+    dark: 'primary',
+  },
+  '--color-subtle': {
+    light: 'slate.950',
+    dark: 'slate.200',
+  },
+  '--color-brand': {
+    light: 'indigo.800',
+    dark: 'indigo.500',
+  },
+  '--color-interactive': {
+    light: 'indigo.600',
+    dark: 'indigo.500',
+  },
+  '--color-interactive-hover': {
+    light: 'indigo.700',
+    dark: 'indigo.600',
+  },
+  '--color-interactive-error': {
+    light: 'red.600',
+    dark: 'red.400',
+  },
+  '--color-interactive-active': {
+    light: 'indigo.600',
+    dark: 'indigo.500',
+  },
+  '--color-warning': {
+    light: 'yellow.100',
+    dark: 'yellow.950',
+  },
+  '--color-text-black': {
+    light: 'black',
+    dark: 'black',
+  },
+  '--color-text-primary': {
+    light: 'primary',
+    dark: 'offWhite',
+  },
+  '--color-text-secondary': {
+    light: 'secondary',
+    dark: 'slate.200',
+  },
+  '--color-text-inverse': {
+    light: 'offWhite',
+    dark: 'primary',
```

Comment:
Looks like we didn't have a dark option for `--color-text-inverse` before, so it was always defaulting to `offWhite` ⬇️ 

```suggestion
    dark: 'offWhite',
```